monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -381,6 +374,9 @@ class ReportPageComponent extends AuthComponent {
   getPotentialSecurityIssuesOverviews() {
     let overviews = [];
     let issues = this.state.report.overview.issues;
+    let state_issues = this.state.issues;
+    issues.push(...state_issues);
+    issues = [...new Set(issues)];
```

Comment:
I'm not sure what's happening here. Why do we need this change?

Reply:
Well we were adding issues on the backend and sending them to the frontend. Now with the credentials endpoints we need to have that functionality on the frontend. I have added issues to the state and I am checking the credentials endpoints to see if I need to add any issues and make sure those are unique.